files/build_templates/docker_image_ctl.j2

```diff
@@ -180,6 +180,9 @@ function postStartAction()
                    ip link add name ns-eth1"$NET_NS" type veth peer name eth1@"$NET_NS"
                    ip link set dev eth1@"$NET_NS" master br1
                    ip link set dev eth1@"$NET_NS" up
+                   # For chassis system where Linux bridge is used on supervisor for midplane communication
+                   # assign alternate name as eth1-midplane for generic design
+                   ip link property add dev br1 altname eth1-midplane
```

Comment:
do you need to check if br1 exists before adding alias to br1?

Reply:
`br` is created by the platform code. In that matter `eth1-midplane` is also created by vendor platform code. In the code above here there is already assumption `br1` is there since we are attaching all the Fabric ASIC Namespace to the `br1` for allowing them to talk to chassis db which is running over `br1` network.  Please note this all logic only applies to Arista Chassis as they have define `midplane_subnet`: https://github.com/sonic-net/sonic-buildimage/blob/a8af76f108893cc04fb472f4537bfc220626c5ef/device/arista/x86_64-arista_7800_sup/chassisdb.conf#L3.  Also since they are using 127.xx.xx.xx network for `br1` they had to create eth1 in Fabric namespace and enable local routing to talk to `br1`. All other vendor fabric namespace reaches chassis db over the docker bridge connected to `eth0` in each fabric namespace.